src/clj/rems/config.clj

```diff
@@ -27,13 +29,17 @@
     config))
 
 (defn- parse-config [config]
-  (update config :email-retry-period #(Period/parse %)))
+  (update config :email-retry-period #(Period/parse %))
```

Comment:
Hmm, aren't the changes done on this line lost after the next line is evaluated?

Reply:
lol, amateur mistake, thanks!

Reply:
:)